### PR TITLE
Fix CS

### PR DESCRIPTION
**Pull request by theofidry**

### CODE REVIEW

File: tests/phpunit/Metrics/MinMsiCheckFailedTest.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function test_it_can_be_created_for_min_MSI(): void
         );
     }
 
-    public function test_it_can_be_created_for_min_covered_code_MSI(): void
+    public function test_it_can_be_created_for_min_covered_code__msi(): void
```
Comment (sanmai) [inline tests/phpunit/Metrics/MinMsiCheckFailedTest.php:53]: These changes seems to a bit wild.

File: tests/phpunit/Metrics/MinMsiCheckerTest.php [hunk 2]
```diff
@@ -156,7 +156,7 @@ public function test_it_does_nothing_if_the_scores_barely_passes_with_no_mutatio
         $msiChecker->checkMetrics(0, 10.2, 10.2, $this->output);
     }
 
-    public function test_it_does_nothing_if_the_MSIs_are_too_low_but_we_ignore_it_with_no_mutations_and_there_is_no_mutations(): void
+    public function test_it_does_nothing_if_the__ms_is_are_too_low_but_we_ignore_it_with_no_mutations_and_there_is_no_mutations(): void
```
Comment (maks-rafalko) [inline tests/phpunit/Metrics/MinMsiCheckerTest.php:159]:
```suggestion
    public function test_it_does_nothing_if_the_msis_are_too_low_but_we_ignore_it_with_no_mutations_and_there_is_no_mutations(): void
```

File: tests/phpunit/TestFramework/Coverage/CoverageCheckerTest.php [hunk 3]
```diff
@@ -196,7 +196,7 @@ public function test_it_does_not_pass_existence_check_if_XML_index_is_missing_wi
         }
     }
 
-    public function test_it_does_not_pass_existence_check_if_XML_index_is_missing_with_PHPUnit(): void
+    public function test_it_does_not_pass_existence_check_if_xml_index_is_missing_with__php_unit(): void
```
Comment (maks-rafalko) [inline tests/phpunit/TestFramework/Coverage/CoverageCheckerTest.php:199]:
```suggestion
    public function test_it_does_not_pass_existence_check_if_xml_index_is_missing_with_php_unit(): void
```

File: src/PhpParser/Visitor/MutationCollectorVisitor.php [hunk 4]
```diff
@@ -45,9 +45,6 @@
  */
 final class MutationCollectorVisitor extends NodeVisitorAbstract
 {
-    /**
-     * @var iterable<Mutation>[]
-     */
```
Comment (maks-rafalko) [inline src/PhpParser/Visitor/MutationCollectorVisitor.php:50]: have you understood why the fixer removes it?

File: tests/phpunit/Metrics/MinMsiCheckFailedTest.php (hunk 1 above)
Comment (theofidry) [reply tests/phpunit/Metrics/MinMsiCheckFailedTest.php]: yeah not a big fan but also not really willing to fight against PHP-CS-Fixer either tbh

File: src/PhpParser/Visitor/MutationCollectorVisitor.php (hunk 4 above)
Comment (theofidry) [reply src/PhpParser/Visitor/MutationCollectorVisitor.php:50]: I think it interpreted it as `mixed[]` and resulted to a "superfluous phpdoc", I changed to to `array<...>` and it seems to do the trick

File: tests/phpunit/Metrics/MinMsiCheckFailedTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (theofidry) [reply tests/phpunit/Metrics/MinMsiCheckFailedTest.php]: thanks, appears I missed a few more when reviewing locally

**Pull request merged**